db/components.go

```diff
@@ -0,0 +1,115 @@
+package db
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/flanksource/commons/logger"
+	"github.com/flanksource/incident-commander/api"
+	"github.com/flanksource/incident-commander/db/types"
+	"github.com/flanksource/incident-commander/utils"
+	"github.com/google/uuid"
+	"gorm.io/gorm/clause"
+)
+
+func GetTeamsWithComponentSelector() map[uuid.UUID][]api.ComponentSelector {
```

Comment:
Lets move this to a `teams` package?

Reply:
Created a teams package for the non-db related things. thanks!